### ISSUE TITLE
Could you please add the way to deal with rewards with same steps in a multi processes training? 

### ISSUE THREAD

**Issue by qiuruiyu** [gxywy/rl-plotter#3]
In my training process,  I use a multi-processes PPO. 
When I want to draw the reward curve with rl-plotter, I found that:
<img width="484" alt="image" src="https://github.com/gxywy/rl-plotter/assets/55018261/a083415f-cbae-4f56-9c2c-da16c739dbea">
Just like the image, there are rewards with the same steps. **But it seems that it only shows one point in the curve?**

**Comment by Xiong5Heng** [comment]
Hi, I also meet the same challenge, have you solved it?

**Comment by qiuruiyu** [comment]
您好，感谢您的来信。已确认收到您的邮件，我会尽快处理您的邮件，谢谢。Hello！This is an automatic reply confirming that your email was received. Your email will be handled as soon as possible. Thank you！                                                      仇睿瑜                                              Joseph QIUThis is an automatic reply, confirming that your e-mail was received.Thank you

**Comment by qiuruiyu** [comment]
> Hi, I also meet the same challenge, have you solved it?

No... I didn't manage to solve the problem because it seems that the curve looks no problem..
